### PR TITLE
feat(seeds-color): add neutral 1100 color

### PR DESCRIPTION
**Pull request by chasemccoy**

### CODE REVIEW

File: packets/seeds-color/tokens.json [hunk 1]
```diff
@@ -137,7 +137,8 @@
       "700" : { "value": "#515e5f" },
       "800" : { "value": "#364141" },
       "900" : { "value": "#273333" },
-      "1000" : { "value": "#162020" }
+			"1000" : { "value": "#162020" },
```
Comment (conwaydev) [inline packets/seeds-color/tokens.json]: nitpicky, but the formatting changed here

Comment (chasemccoy) [reply packets/seeds-color/tokens.json]: good catch

**Comment by krestaino** [comment]
Not sure if this is outside the scope of this PR, or if this is something Marketing Dev should add, but I think this new color should be added here too.

https://github.com/sproutsocial/seeds-packets/blob/develop/packets/seedlings/src/axioms/Colors.scss#L171-L172

@conwaydev Correct me if I'm wrong here.

**Comment by chasemccoy** [comment]
@krestaino Good catch! Feel free to add that if you need it — this new color is something we have to have for dark mode work that's going on. Y'all might not need it, which is totally fine.

**Pull request merged**